### PR TITLE
general structure is added to the HTML

### PR DESCRIPTION
**Pull request by HaniKah**
so guys  ! i have added the general structure to the HTML and i will explain to you on the google meet :)

### CODE REVIEW

**Review by mrkyns — COMMENTED** [review]
Addinig HTML structure.

**Comment by HaniKah** [comment]
and now ?

File: index.html [hunk 1]
```diff
@@ -5,7 +5,14 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="css/style.css" />
-    <title>Document</title>
+    <title>Spatium</title>
```
Comment (mrkyns) [inline index.html:8]: Accept change

**Review by nagonad — APPROVED** [review]
I am trying to approve

File: index.html (hunk 1 above)
Comment (HaniKah) [inline index.html:8]: change the title of the document

**Pull request merged**